content/02.introduction.md

```diff
@@ -1,24 +1,27 @@
-This graph allows for the capacity to make semantic interpretation, continuously incorporate new information and uncover novel hidden knowledge through computational techniques and algorithms.
-Based on this definition resources like Hetionet [@doi:10.7554/eLife.26726] would be considered a knowledge graph as Hetionet integrates multiple sources of information into the form of a graph (example shown in Figure {@fig:hetionet_schema}).
-Hetionet was used to derive novel information concerning unique drug treatments [@doi:10.7554/eLife.26726].
+For this review, we define a knowledge graph as the following: a resource that integrates single or multiple sources of information into the form a graph where nodes repesent entities and edges represent relationships that are shared between two entities.
+This definition is consistent with other definitions found in the literature [@doi:10.3233/SW-160218;10.1109/MIC.2019.2928449;@doi:10.1016/j.eswa.2019.112948;@doi:10.1109/MSN.2016.030;10.1109/jproc.2015.2483592;@doi:10.1145/1242572.1242667;@doi:10.1109/TKDE.2017.2754499].
+Often relationships are considered unidirectional (e.g., a compound treats a disease but a disease cannot treat a compound); however, there are cases where relationships can be considered bidirectional (e.g., a compound resembles another compound or a gene covaries with another gene).
+Based on our definition of a knowledge graph, graphs that we consider knowledge graphs may not be suitable for applications such as symbolic reasoning [@doi:10.1007/978-0-585-29599-2_11] but are still applicable for many biomedical applications.
+Overall, knowledge graphs provide a practical means of understanding complex concepts, continuously incorporating new sources of information and uncovering new knowledge through computational techniques and algorithms. 
+Resources like Hetionet [@doi:10.7554/eLife.26726] would be considered a knowledge graph as Hetionet integrates multiple sources of information into the form of a graph (example shown in Figure {@fig:hetionet_schema}).
+Plus, Hetionet has been used to derive novel information concerning unique drug treatments [@doi:10.7554/eLife.26726].
 We do not consider databases like DISEASES [@doi:10.1016/j.ymeth.2014.11.020] and DrugBank [@doi:10.1093/nar/gkx1037] to be knowledge graphs.
 Although these resources contain essential information, they do not represent their data in the form of a graph.
 
 Knowledge graphs are often constructed from manually curated databases [@doi:10.7554/eLife.26726; 10.1371/journal.pcbi.1002574; @doi:10.1038/s41467-017-00680-8; @doi:10.1186/s12859-016-1336-7;  @doi:10.1016/j.jbi.2011.11.017].
 These databases provide previously established information that can be incorporated into a graph.
 For example, a graph using DISEASES [@doi:10.1016/j.ymeth.2014.11.020] as a resource would have genes and diseases as nodes, while edges added between nodes would represent an association between a gene and a disease.
```

Comment:
Would this be a biomedical knowledge graph?

Reply:
According to our definition, this hypothetical graph would be considered a biomedical knowledge graph.

Reply:
Ok - that was my read too! If you think that's ok, I'm fine with it. I just wanted to make sure I was reading the definition correctly.